integrations/doubleclick-floodlight/lib/index.js

```diff
@@ -28,6 +42,28 @@ var Floodlight = module.exports = integration('DoubleClick Floodlight')
  */
 
 Floodlight.prototype.initialize = function() {
+  var cookieOptions = {
+    // 2 weeks.
+    maxage: 1209600,
+    secure: false,
+    path: '/'
+  };
+  // In the initialize method:
+  // Check if we should load the DoubleClick ID pixel (and only proceed if we haven't already done so).
+  if (this.options.getDoubleClickId && this.options.googleNetworkId) {
+    if (!cookie('doubleclick_id_ts')) {
+      // Load the doubleclick pixel.
+      this.load('doubleclick id', {
+        googleNetworkId: this.options.googleNetworkId,
+        segmentWriteKey: this.options.segmentWriteKey,
+        // TODO: handle userId being nulls/undefined.
```

Comment:
@f2prateek - what would the concern here be if `userId` was undefined? Should this be something that's handled on Personas end?

Reply:
possibly - also wasn't sure how loading the pixel would be handled with a null parameter